ocrd_utils/ocrd_utils/logging.py

```diff
@@ -104,15 +106,15 @@ def initLogging():
 
     logging.basicConfig(
         level=logging.INFO,
-        format='%(asctime)s.%(msecs)03d %(levelname)s %(name)s - %(message)s',
-        datefmt='%H:%M:%S')
+        format=LOG_FORMAT,
```

Comment:
I am confused why this worked in the first place. The `datefmt` didn't include the `year-month-day` part and `%(asctime)` should be formatted according to `datefmt`, yet it was formated `%Y-%m-%d  %H:%m:%s` as it is supposed to. Weird.

Reply:
The reason I got confused is that I didn't read the specs even though @VolkerHartmann reminded me that it was my idea in the first place NOT to have the (redundant) date in the log. So, another chapter in the logging PR saga ....